low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -138,21 +141,23 @@ class SSPLBundleGenerate(object):
                         try:
                             tar.add(key, arcname=value, exclude=lambda x: x in EXCLUDE_FILES)
                         except IOError as err:
-                            print("Unable to include %s logs with an error %s" % (key, err))
+                            self._logger.error("Unable to include %s logs with an error %s" 
+                                               % (key, err))
                 tar.close()
```

Comment:
We can remove this line `tar.close()` as it is unnecessary. This is taken care by context manager.

Reply:
Done